packages/ui/src/components/va-select/components/VaSelectContent/VaSelectContent.vue

```diff
@@ -108,7 +108,7 @@ export default defineComponent({
   components: { VaBadge, VaIcon },
 
   props: {
-    value: { type: Array as PropType<SelectOption[]> },
+    value: { type: Array as PropType<SelectOption[]>, default: [] as SelectOption[] },
```

Comment:
Mustn't it be handled with useStateful composable? Even though, if it is internal component, I would better use `required: true` here.